app/util/confluence/index-snapshot.sh

```diff
@@ -17,7 +17,7 @@ FAIL_FAST_ATTEMPTS=15
 
 
 while [ ${COUNTER} -lt ${ATTEMPTS} ];do
-    if sudo su -c "test -f ${SNAPSHOT}"; then
+    if sudo su -c "test -f ${SNAPSHOT} && find ${SNAPSHOT} -type f -size +${MIN_SNAPSHOT_SIZE} 2>/dev/null | grep -q ."; then
```

Comment:
Maybe it would be better to have simpler logic with getting size of file and if statement?